app/views/events/participants.html.erb

```diff
@@ -1,40 +1,66 @@
 <%- model_class = Event -%>
+<script type="text/javascript">
```

Comment:
I have found onclick to work best as compromise between turbolinks, testing with capybara and readability, e.g. like this: https://gist.github.com/tom95/d29c5488c5e0332f6d95f766a8bc7146